MovieApp/MovieListViewController.swift

```diff
@@ -9,31 +9,65 @@ import Foundation
 import SnapKit
 import UIKit
 import MovieAppData
+import Network
 
 
 protocol SearchBoxActions: AnyObject {
```

Comment:
I would move this delegate (and renamed it to delegate) to a class whose delegate this is, and that is `SearchBarView`.